public/content/documentation/how-to-test/components/links-&-buttons.md

```diff
@@ -24,98 +24,105 @@ best practices.
 
       <div class="how-to-test-checklist-item">
         <h3>✓ Ensure each link and button receives focus and has a visible focus indicator</h3>
```

Comment:
We're missing some headings above this one (from an earlier commit it seems), such as "Test with your keyboard first" and "Test with screen readers second" should be H3s, and "What to test for" should be an H2 ahead of this particular line/heading.

Reply:
hmm I don't think they were headings before. Not seeing that on current day https://karberes.github.io/MagentaA11yV2/#/how-to-test-criteria/components/links-&-buttons 

let me know if I am missing something!

Reply:
I was comparing it to our previous magentaa11y, my bad!